plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -12,7 +12,6 @@
           <and>
             <requires facet="jst.web" version="2.5"/>
             <requires facet="java" version="1.7"/>
-            <conflicts facet="com.google.cloud.tools.eclipse.appengine.facets.flex"/>
```

Comment:
Using comments as weak version control is an antipattern, IMHO. We should move the flex work to a separate branch until we're ready to pull it into head.